tasks/lib/notice/__tests__/notice.js

```diff
@@ -9,11 +9,11 @@ const NODE_MODULES = resolve(__dirname, '../../../../node_modules');
 const NODE_DIR = resolve(process.execPath, '../..');
 const PACKAGES = [
   {
-    name: '@elastic/httpolyglot',
-    version: '0.1.2-elasticpatch1',
+    name: '@spalger/filesaver',
```

Comment:
Okey if I do it in a separate PR? (Just want to make sure this gets ready for 6.0)